pkg/service/HelmAppServiceWrapper.go

```diff
@@ -30,6 +30,17 @@ func NewApplicationServiceServerImpl(logger *zap.SugaredLogger, chartRepositoryL
 	}
 }
 
+func (impl *ApplicationServiceServerImpl) HelmInstallCustom(ctx context.Context, req *client.HelmInstallCustomRequest) (*client.HelmInstallCustomResponse, error) {
```

Comment:
function can be renamed from HelmInstallCustom to InstallReleaseWithCustomChart

Reply:
done